log4j-core/src/test/java/org/apache/logging/log4j/core/pattern/ExtendedThrowablePatternConverterTest.java

```diff
@@ -132,6 +132,55 @@ public void testFiltering() {
         assertTrue(result.contains(" suppressed "), "No suppressed lines");
     }
 
+    @Test
+    public void testFiltersStartFrames() {
```

Comment:
This test indeed checks that the given filter pattern should not be found in the output. Though I would also check if the filter is not provided, the pattern indeed can be found.